### ISSUE TITLE
Feature request: Exclude job posts from Top stories view

### ISSUE THREAD

**Issue by williamtheaker** [Mosc/Glider#45]
I don't want to see job ads every time I look at Hacker News. Adding an option to filter out items where type == `job` would accomplish that.
Example:
https://hacker-news.firebaseio.com/v0/item/30590250.json?print=pretty
```
{
  "by" : "kennandavison",
  "id" : 30590250,
  "score" : 1,
  "time" : 1646672453,
  "title" : "Skio (Subscriptions for Shopify) is hiring founding engineers in NYC",
  "type" : "job",
  "url" : "https://skio.com/careers/"
}
```

**Comment by Mosc** [comment]
Definitely have considered this before, but decided that Hacker News' ads are a fairly unintrusive way of supporting the website compared to other websites' monetization tactics, so I didn't feel comfortable just filtering them out. Opting out of seeing them in the settings might be a decent compromise.

**Comment by Mosc** [comment]
This has been implemented in c2b5d874ea70a5ab8dbc96d40ff8efba999507a5 and should be part of the next release.
